dll/win32/syssetup/lang/hr-HR.rc

```diff
@@ -0,0 +1,264 @@
+    LTEXT "&Administratorska lozinka:", IDC_STATIC, 54, 104, 105, 8
+    EDITTEXT IDC_ADMINPASSWORD1, 165, 101, 148, 14, WS_VISIBLE | WS_TABSTOP | ES_PASSWORD
+    LTEXT "&Potvrdi lozinku:", IDC_STATIC, 54, 123, 105, 8
+    EDITTEXT IDC_ADMINPASSWORD2, 165, 120, 148, 14, WS_VISIBLE | WS_TABSTOP | ES_PASSWORD
+END
+
+IDD_LOCALEPAGE DIALOGEX 0, 0, 317, 143
+STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
+CAPTION "ReactOS Instalacija"
+FONT 8, "MS Shell Dlg"
+BEGIN
+    ICON IDI_ICON4, IDC_STATIC, 21, 7, 20, 20
+    LTEXT "Lokalizacija sustava trebala bi odgovarati jeziku aplikacija koje želite koristiti. Lokalizacija korisnika kontrolira kako se pojavljuju brojevi, valute i datumi.", IDC_STATIC, 53, 7, 253, 20
+    LTEXT "%s", IDC_LOCALETEXT, 53, 29, 250, 16
+    LTEXT "Za promjenu postavki sustava ili korisničkog jezika kliknite Prilagodi.", IDC_STATIC, 53, 60, 196, 8
+    PUSHBUTTON "&Prilagodi...", IDC_CUSTOMLOCALE, 250, 57, 50, 14
+    LTEXT "", IDC_STATIC, 53, 79, 246, 1, SS_ETCHEDHORZ
+    LTEXT "Raspored tipkovnice kontrolira znakove koji se pojavljuju kada tipkate.", IDC_STATIC, 53, 86, 253, 8
+    LTEXT "%s", IDC_LAYOUTTEXT, 53, 100, 250, 16
+    LTEXT "Za promjenu rasporeda tipkovnice kliknite Prilagodi.", IDC_STATIC, 53, 126, 184, 8
+    PUSHBUTTON "Pri&lagodi...", IDC_CUSTOMLAYOUT, 250, 122, 50, 14
+END
+
+IDD_DATETIMEPAGE DIALOGEX 0, 0, 317, 143
+STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
+CAPTION "ReactOS Instalacija"
+FONT 8, "MS Shell Dlg"
+BEGIN
+    ICON IDI_ICON3, IDC_STATIC, 21, 10, 20, 20
+    LTEXT "Datum i vrijeme:", IDC_STATIC, 53, 7, 253, 8
+    CONTROL "", IDC_DATEPICKER, "SysDateTimePick32", DTS_LONGDATEFORMAT | WS_CHILD | WS_VISIBLE | WS_TABSTOP, 53, 17, 137, 14
+    CONTROL "", IDC_TIMEPICKER, "SysDateTimePick32", DTS_TIMEFORMAT | WS_CHILD | WS_VISIBLE | WS_TABSTOP, 200, 17, 54, 14
+    LTEXT "Vremenska zona:", IDC_STATIC, 53, 42, 253, 8
+    COMBOBOX IDC_TIMEZONELIST, 53, 52, 201, 93, CBS_DROPDOWNLIST | CBS_NOINTEGRALHEIGHT | WS_CHILD | WS_VISIBLE | WS_TABSTOP | WS_VSCROLL
+    AUTOCHECKBOX "Automatski namjestite &sat za promjene ljetnog računanja vremena", IDC_AUTODAYLIGHT, 53, 124, 201, 10
+END
+
+IDD_THEMEPAGE DIALOGEX 0, 0, 317, 143
+STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
+CAPTION "ReactOS Instalacija"
+FONT 8, "MS Shell Dlg"
+BEGIN
+    CONTROL "", IDC_THEMEPICKER, "SysListView32", LVS_SINGLESEL | LVS_SHOWSELALWAYS | LVS_ALIGNLEFT | WS_TABSTOP, 18, 8, 288, 130
+END
+
+IDD_PROCESSPAGE DIALOGEX 0, 0, 317, 143
+STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
+CAPTION "ReactOS Instalacija"
+FONT 8, "MS Shell Dlg"
+BEGIN
+    LTEXT "", IDC_ACTIVITY, 53, 7, 253, 20
+    LTEXT "", IDC_ITEM, 53, 29, 253, 20
+    CONTROL "", IDC_PROCESSPROGRESS, "msctls_progress32", PBS_SMOOTH | WS_CHILD | WS_VISIBLE | WS_BORDER, 53, 70, 253, 8
```

Comment:
The build error here is because it's not in sync with `en-US.rc` which was modified since then.